plugins/poll/app/controllers/polls_controller.rb

```diff
@@ -53,26 +53,40 @@ def voters
     poll = Poll.find_by(post_id: post_id, name: poll_name)
     raise Discourse::InvalidParameters.new(:poll_name) if !poll&.can_see_voters?(current_user)
 
-    render json: { voters: DiscoursePoll::Poll.serialized_voters(poll, opts) }
+    if poll.type == "irv"
+      render json: {
+               error: "Invalid poll type, IRV does not support listing voters.",
```

Comment:
we need to have I18N entries for any text

Reply:
I've updated this code to support this method, so this error response has been removed.